www/script/project/rtBuilders.js

```diff
@@ -157,7 +120,8 @@ define(function (require) {
 
             tags.clear();
             $.each(data, function eachBuilder(i, builder) {
-                tags = tags.add(rtBuilders.formatTags(builder.tags, branch_type));
+                var builderTags = rtBuilders.formatTags(builder.tags, branch_type);
+                tags = tags.add(builderTags);
```

Comment:
Is this really doing the right thing? I would assume just tags.add(builderTags) is the right thing?